x-pack/test/cloud_security_posture_functional/pages/cis_integrations/cspm/cis_integration_aws.ts

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../../../ftr_provider_context';
+
+const CIS_AWS_OPTION_TEST_ID = 'cisAwsTestId';
+const AWS_SINGLE_ACCOUNT_TEST_ID = 'awsSingleTestId';
+const AWS_MANUAL_TEST_ID = 'aws-manual-setup-option';
+const AWS_CREDENTIAL_SELECTOR = 'aws-credentials-type-selector';
+const ROLE_ARN_TEST_ID = 'roleArnInput';
+const DIRECT_ACCESS_KEY_ID_TEST_ID = 'directAccessKeyId';
+const DIRECT_ACCESS_SECRET_KEY_TEST_ID = 'directAccessSecretKey';
+const TEMP_ACCESS_KEY_ID_TEST_ID = 'temporaryKeysAccessKeyId';
+const TEMP_ACCESS_KEY_SECRET_KEY_TEST_ID = 'temporaryKeysSecretAccessKey';
+const TEMP_ACCESS_SESSION_TOKEN_TEST_ID = 'temporaryKeysSessionToken';
+const SHARED_CREDENTIALS_FILE_TEST_ID = 'sharedCredentialFile';
+const SHARED_CREDETIALS_PROFILE_NAME_TEST_ID = 'credentialProfileName';
+
+// eslint-disable-next-line import/no-default-export
+export default function (providerContext: FtrProviderContext) {
+  const { getPageObjects, getService } = providerContext;
+  const pageObjects = getPageObjects(['cloudPostureDashboard', 'cisAddIntegration', 'header']);
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Test adding Cloud Security Posture Integrations', function () {
+    this.tags(['cloud_security_posture_cis_integration_cspm_aws']);
+    let cisIntegrationAws: typeof pageObjects.cisAddIntegration.cisAws;
+    let cisIntegration: typeof pageObjects.cisAddIntegration;
+
+    beforeEach(async () => {
+      cisIntegration = pageObjects.cisAddIntegration;
+      cisIntegrationAws = pageObjects.cisAddIntegration.cisAws;
+
+      await cisIntegration.navigateToAddIntegrationCspmPage();
+    });
+
+    after(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+    });
+
+    describe('CIS_AWS Organization Cloud Formation', () => {
+      it('Initial form state, AWS Org account, and CloudFormation should be selected by default', async () => {
+        expect((await cisIntegration.isRadioButtonChecked('cloudbeat/cis_aws')) === true);
+        expect((await cisIntegration.isRadioButtonChecked('organization-account')) === true);
+        expect((await cisIntegration.isRadioButtonChecked('cloud_formation')) === true);
+      });
+      it('Hyperlink on PostInstallation Modal should have the correct URL', async () => {
+        await cisIntegration.clickOptionButton(CIS_AWS_OPTION_TEST_ID);
```

Comment:
this test relies on the result of the prev one. pls check all `it` statements that they can be run independently